cozumler/tekrar-eden-harfleri-bulmaca/java/Cozum.java

```diff
@@ -0,0 +1,25 @@
+import java.util.HashMap;
+import java.util.Map;
+
+public class Cozum {
+
+    public static void main(String[] args) {
+
+        String sentence = "Bir cümle içerisinde tekrar eden harfleri bulmaca";
+        Map<Character, Integer> charMap = new HashMap<>();
+
+        // Karmaşıklık: O(n)
+        for (int i = 0; i < sentence.length(); i++) {
+            char ch = Character.toLowerCase(sentence.charAt(i));
+            if (ch != ' ') {
```

Comment:
biraz gec bir yorum fakat bunlarin yerine for dongusu disarisinda trim ile bosluklari kapatip lower case e donustursen daha guzel olmaz mi?

if burada gereksiz oluyor bu durumda